applications/HDF5Application/custom_io/hdf5_vertex_container_io.cpp

```diff
@@ -10,189 +10,122 @@
 //  Main author:     Máté Kelemen
 //
 
-// Internal includes
-#include "hdf5_vertex_container_io.h"
+// Project includes
+#include "utilities/parallel_utilities.h"
+
+// Application includes
 
+// Include base h
+#include "hdf5_vertex_container_io.h"
 
 namespace Kratos
 {
 namespace HDF5
 {
 
-
-VertexContainerIO::VertexContainerIO(Parameters parameters, File::Pointer pFile)
-        : VertexContainerIO::BaseType(
-            VertexContainerIO::FormatParameters(parameters),
-            pFile,
-            VertexContainerIO::GetPath(parameters))
+VertexContainerCoordinateIO::VertexContainerCoordinateIO(
+    Parameters Settings,
+    File::Pointer pFile)
+    : mpFile(pFile)
 {
     KRATOS_TRY
 
-    parameters.AddMissingParameters(VertexContainerIO::GetDefaultParameters());
-
-    KRATOS_CATCH("");
-}
-
-
-Parameters VertexContainerIO::GetDefaultParameters()
-{
-    return Parameters(R"({
-        "prefix"            : "/",
-        "list_of_variables" : []
-    })");
-}
-
-
-Parameters VertexContainerIO::FormatParameters(Parameters parameters)
-{
-    KRATOS_TRY
+    Parameters default_params(R"(
+        {
+            "prefix"           : "",
+            "write_vertex_ids" : false
+        })");
 
-    Parameters output = parameters.Clone();
-    output.AddMissingParameters(VertexContainerIO::GetDefaultParameters());
-    output["prefix"].SetString("");
-    return output;
+    Settings.AddMissingParameters(default_params);
+    mWriteIDs = Settings["write_vertex_ids"].GetBool();
+    mPathPrefix = Settings["prefix"].GetString();
 
     KRATOS_CATCH("");
 }
 
-
-std::string VertexContainerIO::GetPath(Parameters parameters)
-{
-    KRATOS_TRY
-
-    parameters.AddMissingParameters(VertexContainerIO::GetDefaultParameters());
-    return parameters["prefix"].GetString();
-
-    KRATOS_CATCH("");
-}
-
-
-VertexContainerCoordinateIO::VertexContainerCoordinateIO(Parameters parameters,
-                                                         File::Pointer pFile)
-    : VertexContainerIO(VertexContainerCoordinateIO::FormatParameters(parameters), pFile)
+void VertexContainerCoordinateIO::Write(
+    const Detail::VertexContainerType& rVertices,
```

Comment:
Why vertex? Usually we use node

Reply:
I'm to blame for this one. I came up for [`Vertex`](https://github.com/KratosMultiphysics/Kratos/blob/358979aaeb2b2d82bf7dd63992f48e984f6a04fb/applications/HDF5Application/custom_utilities/vertex.h#L31) in the context of point output to distinguish between `Node`s and general points within the mesh.

A `Vertex` is basically a point inside an element that interpolates nodal values.